parameters.json

```diff
@@ -0,0 +1,15 @@
+{
+  "text": "I'm very happy when I'm at home",
+  "features": {
+    "entities": {
+      "emotion": false,
+      "sentiment": false,
+      "limit": 2
+    },
+    "keywords": {
+      "emotion": true,
+      "sentiment": false,
+      "limit": 2
+    }
+  }
+}
```

Comment:
This file come along with the Untitled-1 file, should we delete it as well?

Reply:
As the tutorial from IBM-Watson, they put separately. The same situation Untitled-1.